frontend/website/static/js/leaderboard.js

```diff
@@ -39,7 +58,9 @@ function start() {
         if (values.length) {
           const headers = values.shift();
           const currentWeekElem = document.getElementById("leaderboard-current-week");
+          const currentWeekChall = document.getElementById("challenges-current-week");
           currentWeekElem.textContent = headers[values[0].length - 1];
+          currentWeekChall.textContent = headers[values[0].length - 1];
```

Comment:
Can you switch to `currentWeekChallenge` for this variable?

`elem` is a well-known and commonly used abbreviation for element, but chall is an abbreviation that is not widely used, so just like in the other place, it's probably worth expanding this out.

Reply:
makes sense!